packages/esm-patient-chart-app/src/encounter-tile/components/tile.scss

```diff
@@ -0,0 +1,96 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@openmrs/esm-styleguide/src/vars' as *;
+
+.title {
+  @include type.type-style('productive-heading-03');
+}
+
+.title:after {
+  content: '';
+  display: block;
+  width: layout.$spacing-07;
+  padding-top: layout.$spacing-01;
+  border-bottom: layout.$spacing-03 solid $brand-teal-01;
+}
+
+.cardTitle {
+  display: flex;
+  justify-content: space-between;
+  padding: layout.$spacing-04 0 layout.$spacing-04 layout.$spacing-05;
+}
+
+.tile {
+  padding: layout.$spacing-01 0 layout.$spacing-05 layout.$spacing-05;
+  margin: layout.$spacing-03;
+  border: solid 1px #e0e0e0;
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  background-color: #ffffff;
+}
+
+.tileTitle {
+  @include type.type-style('label-01');
+  display: block;
+}
+
+.columnContainer {
+  margin: 0px 0px layout.$spacing-06 layout.$spacing-06;
+  font-size: 14px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.29;
+  letter-spacing: 0.16px;
+  color: #525252;
+  display: flex;
+  flex-direction: row;
+}
+
+.tileBoxColumn {
+  width: 100%;
+  display: 'flex';
+  flex-direction: 'row';
+}
+
+.tileSubTitle {
+  height: layout.$spacing-05;
+  font-size: 16px;
+  line-height: 1.33;
+  letter-spacing: 0.32px;
```

Comment:
Shouldn't we be using Carbon's typography defaults here and everywhere else in this file?